webscrambles/src/main/kotlin/org/worldcubeassociation/tnoodle/server/webscrambles/wcif/model/ScrambleSet.kt

```diff
@@ -5,7 +5,8 @@ import org.worldcubeassociation.tnoodle.server.webscrambles.wcif.model.extension
 import org.worldcubeassociation.tnoodle.server.webscrambles.wcif.provider.IndexingIdProvider
 
 @Serializable
-data class ScrambleSet(override val id: Int, val scrambles: List<@Serializable(with = Scramble.Companion::class) Scramble>, val extraScrambles: List<@Serializable(with = Scramble.Companion::class) Scramble>, override val extensions: List<Extension> = emptyList()) : ExtensionProvider(), IndexingIdProvider {
+data class ScrambleSet(override val id: Int, val scrambles: List<@Serializable(with = Scramble.Companion::class) Scramble>, val extraScrambles: List<@Serializable(with = Scramble.Companion::class) Scramble>, override val extensions: List<Extension> = emptyList()
```

Comment:
If you line-break this thing at all, please consider line-breaking per field:

```
data class ScrambleSet(
    val foo: String,
    val bar: Boolean,
    val baz: List<Int>
) : InheritedClass()
```

Reply:
Oh yeah I failed to restore the original indentation